loveship/components/inputs/TextArea.tsx

```diff
@@ -1,32 +1,19 @@
 import React from 'react';
 import { withMods } from '@epam/uui-core';
-import { TextArea as uuiTextArea, TextAreaProps } from '@epam/uui-components';
-import colorStyle from '../../assets/styles/scss/loveship-color-vars.scss';
-import { getTextClasses, TextSettings } from '../../helpers/textLayout';
-import * as types from '../types';
-import css from './TextArea.scss';
+import { TextArea as uuiTextArea, TextAreaProps as UuiTextAreaProps, TextAreaMods as UuiTextAreaMods } from "@epam/uui";
 
-export interface TextAreaMods extends types.EditMode, TextSettings {
-    size?: types.ControlSize | '60';
+export interface TextAreaMods {
```

Comment:
These mods already exist in UuiTextAreaProps, so you don't need to duplicate in here.

Reply:
fixed